src/EnergyPlus/SolarShading.cc

```diff
@@ -8519,66 +8519,59 @@ namespace SolarShading {
-                                            IntBeamAbsByShadFac(BackSurfNum) = BOverlap * AbsSolBeamBackEQL(1, CFS(EQLNum).NL) /
-                                                (Surface(BackSurfNum).Area + SurfaceWindow(BackSurfNum).DividerArea);
-                                            BABSZone += BOverlap * AbsSolBeamBackEQL(1, CFS(EQLNum).NL);
-                                        }
-                                        //   Absorbed by the exterior shade layer of back exterior window
-                                        if (CFS(EQLNum).L(1).LTYPE != ltyGLAZE) {
-                                            IntBeamAbsByShadFac(BackSurfNum) =
-                                                BOverlap * AbsSolBeamBackEQL(1, 1) / (Surface(BackSurfNum).Area + SurfaceWindow(BackSurfNum).DividerArea);
-                                            BABSZone += BOverlap * AbsSolBeamBackEQL(1, 1);
-                                        }
+                                    // Interior beam absorptance of glass layers and beam transmittance of back exterior  &
+                                    // or interior window (treates windows with/without shades as defined) for this timestep
+
+                                    // call the ASHWAT fenestration model for beam radiation here
+                                    CalcEQLOpticalProperty(BackSurfNum, isBEAM, AbsSolBeamBackEQL);
+
+                                    EQLNum = Construct(ConstrNumBack).EQLConsPtr;
+                                    AbsBeamWinEQL({ 1, CFS(EQLNum).NL }) = AbsSolBeamBackEQL(1, { 1, CFS(EQLNum).NL });
+                                    // get the interior beam transmitted through back exterior or interior EQL window
+                                    TransBeamWin = AbsSolBeamBackEQL(1, CFS(EQLNum).NL + 1);
+                                    //   Absorbed by the interior shade layer of back exterior window
+                                    if (CFS(EQLNum).L(CFS(EQLNum).NL).LTYPE != ltyGLAZE) {
+                                        IntBeamAbsByShadFac(BackSurfNum) = BOverlap * AbsSolBeamBackEQL(1, CFS(EQLNum).NL) /
+                                            (Surface(BackSurfNum).Area + SurfaceWindow(BackSurfNum).DividerArea);
+                                        BABSZone += BOverlap * AbsSolBeamBackEQL(1, CFS(EQLNum).NL);
+                                    }
+                                    //   Absorbed by the exterior shade layer of back exterior window
+                                    if (CFS(EQLNum).L(1).LTYPE != ltyGLAZE) {
+                                        IntBeamAbsByShadFac(BackSurfNum) =
+                                            BOverlap * AbsSolBeamBackEQL(1, 1) / (Surface(BackSurfNum).Area + SurfaceWindow(BackSurfNum).DividerArea);
+                                        BABSZone += BOverlap * AbsSolBeamBackEQL(1, 1);
+                                    }
 
```

Comment:
This is all just reducing the indent because the surrounding `if` was removed.

Reply:
:+1: